lib/Net/ThreeScale/Client.pm

```diff
@@ -48,7 +48,7 @@ sub new {
 
 	my $self = {};
 	$self->{provider_key} = $params->{provider_key};
-	$self->{url}          = $params->{url} || 'http://su1.3scale.net';
+	$self->{url}          = $params->{url} || 'https://su1.3scale.net';
```

Comment:
We've been running with this change for some time, however we use cacheing and aggregation to minimise our calls to 3scale. Users probably should be using https to contact 3scale, but I agree that it may not be suitable for everyone until we're doing persistent connections.

I'll remove this change from this pull request and leave it for a future change.